libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -98,10 +98,6 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
     @Input()
     noBorder = false;
 
-    /** Whether list component has removed bottom borders */
-    @Input()
-    noSeperator: boolean;
```

Comment:
Breaking change?

Reply:
Thanks. Added breaking change information in Wiki.